### ISSUE TITLE
Newline not added to .gitignore entry on add

### ISSUE THREAD

**Issue by pedroos** [sobolevn/git-secret#643]
What are the steps to reproduce this issue?
-------------------------------------------
1. Add multiple files to git-secret

What happens?
-------------
- The .gitignore entries created don't have newlines between them (thus the entries don't work)

What were you expecting to happen?
----------------------------------
- Newlines between .gitignore entries inserted by git-secret

Any other comments?
-------------------
Here is a fix:

```
diff --git a/src/_utils/_git_secret_tools.sh b/src/_utils/_git_secret_tools.sh
index 5ade310..276dd6d 100644
--- a/src/_utils/_git_secret_tools.sh
+++ b/src/_utils/_git_secret_tools.sh
@@ -335,7 +335,7 @@ function _add_ignored_file {
   local full_path
   full_path=$(_append_root_path '.gitignore')
 
-  printf '%q' "$filename" >> "$full_path"
+  printf '%q\n' "$filename" >> "$full_path"
 }
```

Should I create a pull request?

What versions of software are you using?
----------------------------------------
Master: ee592b8ede9c91b987c9cf6b7e34ab45f2f0f8e5

**Comment by joshrabinowitz** [comment]
Hello, @pedroos 

Thank you very much for reporting this issue, and for your patch. 

We will take a look at it soon and likely integrate your suggestion. If you wanted to submit a PR, that would be welcome too!

**Comment by joshrabinowitz** [comment]
@pedroos I'm planning to merge this soon. If you could add a test showing it works as expected that would be great!

**Comment by pedroos** [comment]
@joshrabinowitz, Hi, sorry, I won't be able to contribute this test.

**Comment by joshrabinowitz** [comment]
Closed by #645
